schema.json

```diff
@@ -95,6 +94,7 @@
           "type": "string",
           "title": "Target-Path Schema",
           "description": "Base destination folder where any archive/fill will be inflated/written. Default is device internal SD-card root",
+          "default": "",

```

Comment:
Is there any json-schema way to actually tie this value to target-path as the description implies?